roomserver/storage/postgres/previous_events_table.go

```diff
@@ -82,22 +82,21 @@ func (s *previousEventStatements) InsertPreviousEvent(
 	ctx context.Context,
 	txn *sql.Tx,
 	previousEventID string,
-	previousEventReferenceSHA256 []byte,
 	eventNID types.EventNID,
 ) error {
 	stmt := sqlutil.TxStmt(txn, s.insertPreviousEventStmt)
 	_, err := stmt.ExecContext(
-		ctx, previousEventID, previousEventReferenceSHA256, int64(eventNID),
+		ctx, previousEventID, []byte(""), int64(eventNID),
```

Comment:
Why not remove `[]byte("")` entirely? I guess removing `previous_reference_sha256` is for another PR? Actually it looks like you just forgot about this, given sqlite does remove this.